src/api/entities/SecurityToken/TokenHolders.ts

```diff
@@ -11,10 +14,22 @@ import { SecurityToken } from './';
  * Handles all Security Token Holders related functionality
  */
 export class TokenHolders extends Namespace<SecurityToken> {
+  public async get(opts: { pagination: PaginationOptions }): Promise<ResultSet<IdentityBalance>>;
+
+  public async get(opts: { mintStatus: boolean }): Promise<ResultSet<IdentityBalance>>;
```

Comment:
These overloads are incorrect.

First, you should be able to pass pagination options regardless of whether you pass other options. Also, the typings should reflect the fact that if you don't pass `mintStatus`, you get an object without the `canMint` property.

So, something like

```
// SecurityToken/types.ts

interface TokenHolderOptions {
  canBeIssuedTo: boolean;
}

interface TokenHolderProperties {
  canBeIssuedTo: boolean;
}

// This file

public async get(opts: Pick<TokenHolderOptions, 'canBeIssuedTo'>, paginationOpts?: PaginationOptions): Promise<ResultSet<IdentityBalance & Pick<TokenHolderProperties, 'canBeIssuedTo'>>>;

public async get(paginationOpts?: PaginationOptions): Promise<ResultSet<IdentityBalance>>;
```